workflow-controller/controller-service/src/main/java/io/hyscale/controller/hooks/VolumeCleanUpHook.java

```diff
@@ -80,6 +80,7 @@ public void postHook(WorkflowContext context) throws HyscaleException {
 		for (Manifest manifest : mainfestList) {
 			try {
```

Comment:
what happens if a new kind deployed through custom snippets has been deleted in the next deployment? will that resource be deleted from the namespace. any changes required to K8SResourcesCleanUpHook?